AUTHORS.md

```diff
@@ -8,5 +8,6 @@ Matthew Drooker <matthew.drooker@gmail.com>
 A.D. Slaton <ad.slaton@turner.com>
 Sam Schenkman-Moore <samuel.schenkman-moore@turner.com>
 Randall Potter <randallbpotter15@gmail.com>
+Brian Duckett <bduckett@codeclout.com>
```

Comment:
This file is updated when we run the contributors/author script. Please undo this.